publish/pipeline-build-deploy-app/pipeline-build-deploy-app.md

```diff
@@ -0,0 +1,167 @@
+   * Run `oc apply nfs-pv.yaml`
+3. If you are using a private GitHub repo, create the secret for your repo. Follow the steps in Secrets section of [Working with Pipelines](../working-with-pipelines/working-with-pipelines.html#getting-started).
+4. Determine whether it is necessary to create the secret for your image registry.
+   * If you are using the internal route of the internal registry of your OCP cluster, you do not have to configure a secret as long as the service account you are running the pipeline with has appropriate permissions, such as the `kabanero-pipeline` service account.
+   * If you are using the external route of the internal registry of your OCP cluster, you do not have to configure a secret. The pipeline will recognize the URL and translate it to the internal route of the registry.
+   * If you are using a private registry, follow the steps in Secrets section of [Working with Pipelines](../working-with-pipelines/working-with-pipelines.html#getting-started) to set up a secret for your registry.
+5. Set up certificates or specify insecure connection for the registry access. For instructions and additional information, see Transport layer security (TLS) verification for image registry access in pipelines in [Working with pipelines](../working-with-pipelines/working-with-pipelines.html#transport-layer-security-tls-verification-for-image-registry-access-in-pipelines).
+6. Run the pipeline. You can run pipelines from the pipelines dashboard with webhook extension, using scripts, or by using the command line. For more information, see the appropriate sections of [Working with pipelines](../working-with-pipelines/working-with-pipelines.html).
+7. Review the status of your run and log files. For more information, see Checking the status of the pipeline run in [Working with pipelines](../working-with-pipelines/working-with-pipelines.html#checking-the-status-of-the-pipeline-run).
+8. When the pipeline run is complete, you can check your registry to see if the application image is published.
+9. If you ran the pipeline using the webhook, it will only deploy the application if you initiated the run via a pull request merge. In that case, continue to the next step to access your application.
+10. To see your application pod running, run `oc get apposodyapplications`, or check your pods.
+11. For private registries and the external route of the internal registry, create the secret for your registry and associate it with the new service account that was created by Appsody for your application. If you are using the internal route for the internal registry, this step is not necessary, because the service account has the necessary permissions. For instructions, see [Creating a secret and linking to a service account](#cslsa).
+12. If you exposed a route for your application, check the routes (for example, run `oc get routes`) and test your application.
+
+
+### Creating a secret and linking to a service account
+{: #cslsa}
```

Comment:
The steps are butting right up to the subtitle here, which doesn't look great. I initially thought it was your curly braces, but it isn't.

BTW why do you not link like this rather than use curlies? Is that basic markdown or an extension they've added to the build. I didn't know you could do that! 

```
[Creating a secret and linking to a service account](#creating-a-secret-and-linking-to-a-service-account)
```

To add the space I just inserted `<br>` under the heading.